core/keyserver/keyserver.go

```diff
@@ -102,8 +103,7 @@ func (s *Server) getEntry(ctx context.Context, userID string, epoch int64) (*tpb
 
 	neighbors, err := s.tree.NeighborsAt(ctx, index[:], epoch)
 	if err != nil {
-		log.Printf("Cannot get neighbors list: %v", err)
```

Comment:
I don't have a preference between exported/unexported and server-module/non-server-module. We can decide on either. The important thing is to have a consistent error generation/handling rule. Currently, `keyserver.go` functions return both `error` and gRPC error objects. The good thing is that gRPC handlers always return gRPC errors.